include/xrpl/protocol/detail/ledger_entries.macro

```diff
@@ -499,6 +499,7 @@ LEDGER_ENTRY(ltVAULT, 0x0084, Vault, vault, ({
     {sfLossUnrealized,       soeREQUIRED},
     {sfShareMPTID,           soeREQUIRED},
     {sfWithdrawalPolicy,     soeREQUIRED},
+    {sfScale,                soeDEFAULT},
```

Comment:
```suggestion
    {sfScale,                soeOPTIONAL},
```
If the default value `sfScale` in the Vault object is `vaultDefaultIOUScale`, it should be `soeOPTIONAL`.  Making it `soeDEFAULT` would mean that this scale intends to have a default value of 0, which is not the case here.